GoogleUtilities/SceneDelegateSwizzler/GULSceneDelegateSwizzler.m

```diff
@@ -78,8 +78,30 @@ + (BOOL)isSceneDelegateProxyEnabled {
   return [GULAppDelegateSwizzler isAppDelegateProxyEnabled];
 }
 
++ (void)proxyOriginalSceneDelegate {
+  if ([GULAppEnvironmentUtil isAppExtension]) {
+    return;
+  }
+
 #if UISCENE_SUPPORTED
```

Comment:
should this check be at line 82 instead? If there is no support for UIScene, the whole method should be a no-op. Otherwise, it will check for the app extension and it can take a few processing cycles.